gspread/models.py

```diff
@@ -1339,6 +1339,43 @@ def insert_row(
 
         return data
 
+        def insert_rows(self, values, index=1, value_input_option='RAW'):
```

Comment:
Thank you for the update. It looks like the indentation of this line is wrong.
Could you please fix this?